src/main/java/nl/tudelft/contextproject/Main.java

```diff
@@ -118,11 +119,16 @@ protected void setInputManager(InputManager im) {
 		inputManager = im;
 	}
 
+
 	@Override
 	public void simpleInitApp() {
+		tickListeners = new LinkedList<>();
 		setDisplayFps(debugHud);
 		setDisplayStatView(debugHud);
-		getFlyByCamera().setMoveSpeed(50);
+		
+		flyCam.setMoveSpeed(100);
+		viewPort.setBackgroundColor(new ColorRGBA(0.1f, 0.1f, 0.1f, 1f));

```

Comment:
If you are certain the VR camera won't use this I'm fine with it staying like this.